shotover-proxy/examples/cassandra-protect-local/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.3"
+services:
+  cassandra-one:
+    image: library/cassandra:3.11.10
+    ports:
+      - "9043:9042"
+    environment:
+      MAX_HEAP_SIZE: "400M"
+      MIN_HEAP_SIZE: "400M"
+      HEAP_NEWSIZE: "48M"
+      CASSANDRA_ENABLE_SCRIPTED_USER_DEFINED_FUNCTIONS: "true"
```

Comment:
Don't need this if you aren't using any UDFs

Reply:
I would like to run the full test suite in these tests eventually, but for now i've left the rest of the test suite out to make developing Protect easier.
Maybe I should just add the rest of the test suite now though?

Reply:
Sure, might as well include in this PR.

Reply:
alright, added